Documentation/developer-guide.md

```diff
@@ -23,58 +23,61 @@ Please read [Contributing](https://github.com/dotnet/wcf/blob/master/Documentati
-more running WCF services.  By default, they are not run with the normal build.cmd.
-To run the scenario tests, execute this CMD from the root of the repository:
+Scenario tests are those tests that involve network activity between the client tests and WCF test services.
+By default, they are not run with the normal build.cmd or build.sh.
+
+To run the scenario tests on Windows, execute this CMD from the root of the repository. It will automatically start a
+self-hosted WCF test service on the same Windows machine and shut it down at the end of execution of all tests.
 
 ```
     build.cmd /p:WithCategories=OuterLoop
 ```
+To run scenario tests against an already running WCF test service, you will need to specify the base address 
+of the WCF test service by using the `/p:ServiceUri` parameter. This is also necessary when running
+scenrio tests on Linux or OS X as the WCF service can only run on Windows.
+
+From the root of the repository, run this command to run scenario tests on Windows:
+```
+    build.cmd /p:WithCategories=OuterLoop /p:ServiceUri=[WCFTestServiceBaseAddress]
+```
+or run this command to run scenario tests on Linux or OS X.
+Note some of security related tests require extra manual setup to work, 
+see [Linux manual test guide](https://github.com/dotnet/wcf/blob/master/Documentation/manualtest-guide.md) for details.
+```
+    ./build.sh /p:WithCategories=OuterLoop /p:ServiceUri=[WCFTestServiceBaseAddress]
+```
+
+You will need to replace `[WCFTestServiceBaseAddress]` in above commands with an actual value.
+See [Scenario test guide](https://github.com/dotnet/wcf/blob/master/Documentation/scenario-test-guide.md)
+for further details including how to start a WCF test service.
 
 Obtaining code coverage
 =======================
 You can also obtain detailed code coverage information by including an additional property
 when you run the build script.  For example, this CMD runs the scenario tests and collects
-code coverage numbers:
+code coverage numbers on Windows:
 ```
     build.cmd /p:WithCategories=OuterLoop /p:Coverage=true
 ```
-
-Running tests with latest dependency versions
-=============================================
-Running tests against each dependency's latest version can catch functional issues. The
-`FloatingTestRuntimeDependencies` property enables this. To run unit tests against the
-latest dependencies:
-```
-    build.cmd /p:FloatingTestRuntimeDependencies=true
-```
-To run the scenario tests with latest dependencies:
-```
-    build.cmd /p:FloatingTestRuntimeDependencies=true /p:WithCategories=OuterLoop
-```
-
-Running scenario tests across multiple machines
-===============================================
-By default scenario tests run against WCF services that are locally self-hosted.
-To run these tests against WCF services running on a separate machine, see:
-[Cross-machine test guide](https://github.com/dotnet/wcf/blob/master/Documentation/cross-machine-test-guide.md).
-
-
-
```

Comment:
Consider: the pitfall where if you change server, might want to point out deleting "Infrastructure.Common" might help